webpack/webpack.config.prod.js

```diff
@@ -0,0 +1,14 @@
+const { merge } = require('webpack-merge');
+const common = require('./webpack.config.common');
+const TerserPlugin = require('terser-webpack-plugin');
+
+module.exports = merge(common, {
```

Comment:
Production build does not use `HtmlWebpackPlugin` so if you build prod build from scratch then you will miss index.html file.

Reply:
fixed